frontend/components/common/Table.tsx

```diff
@@ -303,17 +332,17 @@ const Table = ({
         <h1>No matches were found. Please change your filters.</h1>
       )}
       {pageOptions.length > 1 && (
-        <div className="is-clearfix" style={{ display: 'flex' }}>
+        <div className="is-clearfix">
           <button
             style={{ marginRight: '0.5rem' }}
-            onClick={() => gotoPage(0)}
+            onClick={() => previousPage()}
             disabled={!canPreviousPage}
           >
             <Icon name="chevrons-left" />
```

Comment:
This is the correct behavior, only the right buttons have the behavior swapped.

The single arrow buttons should go forward/back one page, while the double arrow buttons should go all the way to the beginning/end.